docs/DE0009.md

```diff
@@ -0,0 +1,14 @@
+<!--
+P:System.Environment.OSVersion
+-->
+
+# DE0009: Environment.OSVersion shouldn't be used
+
+## Motivation
+
+The `OSVersion` property doesn't provide a reliable way to identify the exact OS and its version.
+
+## Recommendation
+
+Use the method `RuntimeInformation.IsOSPlatform` to identify the OS platform and avoid writing
```

Comment:
We usually say what the API is after: Use the `RuntimeInformation.IsOSPlatform` method...

Link to the API on docs? (https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform)

The last part of this sentence is a bit hard to read. How about this?
Use the `RuntimeInformation.IsOSPlatform` method to identify the OS platform. Avoid writing code dependent on reported OS version and instead check for availability of needed features (ideally you'd say how here).

Reply:
The exact check depends on the feature in question so here it needs to be very generic.

Reply:
That's ok. Just to be super clear, I was more thinking of the technique you'd use to do the check. But  if you can't describe it, it's fine as-is.